kart/tabular/working_copy/table_defs.py

```diff
@@ -262,6 +262,24 @@ def __init__(self):
             ),
         )
 
+    ESPG_4326 = (
+        'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],'
+        'PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]]'
+    )
+
+    @classmethod
+    def init_table_contents(cls, sess):
+        sess.execute(
+            f"""
+            INSERT OR REPLACE INTO gpkg_spatial_ref_sys
+            (srs_name, srs_id, organization, organization_coordsys_id, definition, description)
+            VALUES
+            ('Undefined cartesian SRS', -1, 'NONE', -1, 'undefined', 'undefined cartesian coordinate reference system'),
+            ('Undefined geographic SRS', 0, 'NONE', 0, 'undefined', 'undefined geographic coordinate reference system'),
+            ('WGS 84 geodetic', 4326, 'EPSG', 4326, '{cls.ESPG_4326}', 'longitude/latitude coordinates in decimal degrees on the WGS 84 spheroid')
```

Comment:
i'd suggest parametrizing this properly - no real reason not to

Reply:
Done